### ISSUE TITLE
Focus should be given to toast notifications

### ISSUE THREAD

**Issue by eleanor-heath** [adaptlearning/adapt-contrib-core#51]
Currently screen readers do not announce toast notifications, to increase accessibility coverage we need them to be announced.

**Steps to replicate**

1. Create a course with a multiple choice question
2. Add a few items to the multiple choice question
3. Add the learning objectives extension an add a learning objectives code
4. Add the assisted learning extension and enable
5. Edit the multiple choice question to also include the learning objective
6. Preview the course and answer the multiple choice question correctly
